callforpapers.md

```
@@ -0,0 +1,120 @@
+* **Acceptance decisions: ~~June 5~~ ~~June 9~~ 2026.** The acceptance notification is delayed — we are working on finalizing the reviews, please stay tuned.
+* Authors of spotlighted papers will be invited to give a short talk at the workshop and will be notified with details shortly after acceptance notifications.
+
+DL4C is **non-archival** and thus **dual submission is allowed** where permitted by third parties. Please see the FAQ below for more details.
+
+Authors should check their email (and spam folder) for acceptance decisions and instructions on how to submit their camera-ready paper and poster by the deadline.
+
+---
+
+## FAQ
+
+##### **Will the OpenReview be open to the public?**
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h5

(MD001, heading-increment)